test/Fixtures/CostModels.purs

```diff
@@ -9,179 +9,182 @@ import Data.Tuple.Nested ((/\))
-          [ 197209
-          , 0
-          , 1
-          , 1
-          , 396231
-          , 621
-          , 0
-          , 1
-          , 150000
-          , 1000
-          , 0
-          , 1
-          , 150000
-          , 32
-          , 2477736
-          , 29175
-          , 4
-          , 29773
-          , 100
-          , 29773
-          , 100
-          , 29773
-          , 100
-          , 29773
-          , 100
-          , 29773
-          , 100
-          , 29773
-          , 100
-          , 100
-          , 100
-          , 29773
-          , 100
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 1000
-          , 0
-          , 1
-          , 150000
-          , 32
-          , 150000
-          , 1000
-          , 0
-          , 8
-          , 148000
-          , 425507
-          , 118
-          , 0
-          , 1
-          , 1
-          , 150000
-          , 1000
-          , 0
-          , 8
-          , 150000
-          , 112536
-          , 247
-          , 1
-          , 150000
-          , 10000
-          , 1
-          , 136542
-          , 1326
-          , 1
-          , 1000
-          , 150000
-          , 1000
-          , 1
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 1
-          , 1
-          , 150000
-          , 1
-          , 150000
-          , 4
-          , 103599
-          , 248
-          , 1
-          , 103599
-          , 248
-          , 1
-          , 145276
-          , 1366
-          , 1
-          , 179690
-          , 497
-          , 1
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 148000
-          , 425507
-          , 118
-          , 0
-          , 1
-          , 1
-          , 61516
-          , 11218
-          , 0
-          , 1
-          , 150000
-          , 32
-          , 148000
-          , 425507
-          , 118
-          , 0
-          , 1
-          , 1
-          , 148000
-          , 425507
-          , 118
-          , 0
-          , 1
-          , 1
-          , 2477736
-          , 29175
-          , 4
-          , 0
-          , 82363
-          , 4
-          , 150000
-          , 5000
-          , 0
-          , 1
-          , 150000
-          , 32
-          , 197209
-          , 0
-          , 1
-          , 1
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 150000
-          , 32
-          , 3345831
-          , 1
-          , 1
-          ]
-      )
+  [ PlutusV1 /\ costModelFixture
+  , PlutusV2 /\ costModelFixture
```

Comment:
`PlutusV2` has different cost models

Reply:
I've done most of the work to support PlutusV2 cost models, however I've hit a CSL bug: https://github.com/Emurgo/cardano-serialization-lib/issues/473

Reply:
I think I might still be able to work with it via the bytes interface, however. I'm going to give it a go

Reply:
@ngua Should we update the plutusv1 cost model fixture to the vasil values?

Reply:
Thanks @vsubhuman!

@jy14898 I think we should just remove the PS fixture and use CSL's `TxBuilderConstants.plutus_vasil_cost_models`. We can convert it to our `Costmdls` with `Deserialization.Transaction.convertCostModels`